cartographer_grpc/framework/server.h

```diff
@@ -104,9 +104,10 @@ class Server {
     return {execution_context_->lock(), execution_context_.get()};
   }
 
- private:
+ protected:
```

Comment:
You only need to expose AddService? Maybe leave the rest private.

Reply:
Actually the destructor needs to be public and I need the constructor to be protected as well.

Reply:
Done.